README.md

```diff
@@ -1,17 +1,33 @@
 # fastapi-nextauth-jwt
+
 [![PyPI version](https://badge.fury.io/py/fastapi-nextauth-jwt.svg)](https://badge.fury.io/py/fastapi-nextauth-jwt)
+[![PyPI Downloads](https://img.shields.io/pypi/dm/fastapi-nextauth-jwt)](https://pypi.org/project/fastapi-nextauth-jwt/)
+[![License](https://img.shields.io/pypi/l/fastapi-nextauth-jwt)](https://github.com/yourusername/fastapi-nextauth-jwt/blob/main/LICENSE)
+[![Contributors](https://img.shields.io/github/contributors/TCatshoek/fastapi-nextauth-jwt)](https://github.com/TCatshoek/fastapi-nextauth-jwt/graphs/contributors)
+
+This project provides a FastAPI dependency for decrypting and validating JWTs generated by Auth.js. It is designed to facilitate the integration of a FastAPI backend with Next.js and NextAuth/Auth.js on the frontend.
+
+> [!WARNING]
+> It may also be compatible with other frameworks using Auth.js, though this has not been extensively tested.
+
+> [!IMPORTANT]
+> NextAuth.js has been renamed to Auth.js, and the default cookie names have changed to reflect this (e.g., from `next-auth.csrf-token` to `authjs.csrf-token`). You'll need to update your cookie name configuration accordingly if you're using the latest version of Auth.js.
```

Comment:
This is incorrect, starting from version 2.0.0 the default cookie names take this change into account, see: https://github.com/TCatshoek/fastapi-nextauth-jwt/blob/bef787beff050a4c62a96b721bfbc4d8793d3f33/src/fastapi_nextauth_jwt/fastapi_nextauth_jwt.py#L88